media-production/problem-statement.md

```diff
@@ -0,0 +1,56 @@
+Media Production on the Web
+===========================
+
+Background
+----------
+
+Professional media assets, including audio-visual masters for television and motion pictures, are increasingly being stored in the cloud.
+
+There is a corresponding growing interest in building web applications that allow end-users to manipulate these assets, e.g., quality checking, versioning, timed text authoring, etc.
+While the web platform has evolved to support consumer media applications, professional applications require additional capabilities, including precise timing, wider color gamut and high-dynamic range, high-fidelity timed text, etc.
+
+Problem statement
+-----------------
+
+What are the current technical challenges, and future opportunities, for professional media applications on the Web, and how should they be addressed by the Web Platform?
+
+Potential topics
+----------------
+
+* How can a video editing application reach a specific frame of an arbitrary media stream?
+
+* What is missing to enable color grading within a web application?
+
+* What is missing to enable monitoring visual effects in real time?
+
+* Support for professional video codecs, e.g. JPEG 2000, etc.
+
+* Support for HDR and WCG color spaces, e.g. P3D65, etc.
+
+* In general, media playback on the Web usually favors user experience over fidelity to the initial content. Professional media applications need to run in high-fidelity mode, e.g. for timed text. Can new features or new technologies help in this regard?
+
+* Are there specific formats that professional media applications need to support? Are these formats suitable for use in Web browsers?
+
+* What are common mechanisms used within professional media applications to process media efficiently? Would specific APIs for media processing help?
+
+* What techniques can professional media applications use and what additional capabilities could help to localize media content?
+
+* What capabilities - APIs, semantics, techniques for rendering, processing, personalization, customization, interoperability, etc. - can developers leverage to ensure accessibility of media content authoring tools and of the media content produced?
+
+Stakeholders
+------------
+
+* Post-production tools editors, e.g. linear and non-linear video editing, dubbing, special effects, QC, timed text editing, etc.
+
+* Production and post-production service providers
+
+* Content distributors
+
+* Browser vendors
+
+Contact
+-------
+
+[Chris Needham](chris.needham@bbc.co.uk)
+[Pierre-Anthony Lemieux](pal@sandflow.com)
+[Tatsuya Igarashi](Tatsuya.Igarashi@sony.com)
```

Comment:
I would also invite feedback in public forms (which some developers might favour), as you did in the email you sent to the group:

```
Interested parties are encouraged to get in touch with the group, either through [GitHub issues](https://github.com/w3c/media-and-entertainment/issues) or the group's [public mailing-list](mailto:public-web-and-tv@w3.org). You may also contact the group's chairs or W3C team:

* [Chris Needham](chris.needham@bbc.co.uk) - Chair
* [Pierre-Anthony Lemieux](pal@sandflow.com) - Chair
* [Tatsuya Igarashi](Tatsuya.Igarashi@sony.com) - Chair
* [Kazuyuki Ashimura](ashimura@w3.org) - W3C team
* [Francois Daoust](fd@w3.org) - W3C team
```

Reply:
Thanks, I added this.